tests/components/knx/conftest.py

```diff
@@ -30,13 +30,22 @@
     DOMAIN as KNX_DOMAIN,
 )
 from homeassistant.components.knx.project import STORAGE_KEY as KNX_PROJECT_STORAGE_KEY
+from homeassistant.components.knx.storage.config_store import (
+    STORAGE_KEY as KNX_CONFIG_STORAGE_KEY,
+)
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
 from homeassistant.helpers.typing import ConfigType
 from homeassistant.setup import async_setup_component
 
+from . import KnxEntityGenerator
+
 from tests.common import MockConfigEntry, load_fixture
+from tests.typing import WebSocketGenerator
 
 FIXTURE_PROJECT_DATA = json.loads(load_fixture("project.json", KNX_DOMAIN))
+FIXTURE_CONFIG_STORAGE_DATA = json.loads(load_fixture("config_store.json", KNX_DOMAIN))
```

Comment:
If its an object:
```suggestion
FIXTURE_CONFIG_STORAGE_DATA = load_json_object_fixture("config_store.json", KNX_DOMAIN)
```
if its an array
```suggestion
FIXTURE_CONFIG_STORAGE_DATA = load_json_array_fixture("config_store.json", KNX_DOMAIN)
```

Reply:
👍 applied for the line above too